File: ic/Pairs.cpp
#include <iostream>
using namespace std;

int main(){

int i, min, max, num;
cout<<"Digite dois números para definir o intervalo: ";

cin >> min;
cin >> max;

for (i = min; i <= max; i++) {
int num = i%2;
if (num%2 == 0){

cout << "Numero par:" << i<< "!";
 }
}
return 0;
}

File: ic/Pairs.cs
using System;
public class Program{
public static void Main(){

int i;

Console.WriteLine("Digite dois números para definir o intervalo:");

int min = Convert.ToInt32(Console.ReadLine());
int max = Convert.ToInt32(Console.ReadLine());

for (i = min; i <= max; i++) {
var num = i%2;
if (num%2 == 0){
    
Console.WriteLine("Número par: {i}" + i);
   }
  }
 }
}